src/components/VMarkAllAsRead.vue

```diff
@@ -3,7 +3,7 @@ import { ref } from "vue"
 import { allActivities } from "../composables/pulls"
 import { markAsRead, hasRead } from "../composables/read"
 
-const isCaughtUp = ref(allActivities().map(hasRead).some(Boolean))
+const isCaughtUp = ref(allActivities().map(hasRead).every(Boolean))
```

Comment:
So I think this is the correct logic (whoops :sweat_smile:) but calculating it needs to be deferred until there actually are activities loaded, otherwise (and this is a TIL for me):

```
> [].map(() => {}).every(Boolean)
true
```

Whoops!